package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "music-api",
+  "version": "1.0.0",
+  "description": "Build front end for Music-Api site. This music Api provides music albums for different genres.",
+  "main": "index.js",
+  "scripts": {
+    "dev": "./node_modules/.bin/env-cmd ./node_modules/.bin/nodemon index.js",
+    "dev:migrate": "./node_modules/.bin/env-cmd ./node_modules/.bin/sequelize db:migrate --env development --config ./configs/sequelize.js",
```

Comment:
Your migrations fail. You sequelize config is expecting dialect to be an environment variable but you did not specify that in your .env-sample file. Also your config is expecting `DB_DATABASE` but your .env-sample file is setting `DB_DB_NAME`

![image](https://user-images.githubusercontent.com/5775716/84066660-410f5800-a994-11ea-8a77-ce14a0418b57.png)